Wizardry/Core/BattleSys/Source/PreBattleCalc.c

```diff
@@ -702,6 +702,11 @@ void PreBattleCalcSkills(struct BattleUnit * attacker, struct BattleUnit * defen
     }
 #endif
 
+#if (defined(SID_StunningSmile) && (SID_StunningSmile < MAX_SKILL_NUM))
+    if (SkillTester(&attacker->unit, SID_StunningSmile) && !(UNIT_CATTRIBUTES(&defender->unit) && CA_FEMALE))
+        defender->battleAvoidRate -= 20;
```

Comment:
do not change defender's status. Please be consistent with the function of the vanilla function.

Reply:
An equivalent alternative is what the original skill system does AliScrooge.

![image](https://github.com/MokhaLeee/fe8u-cskillsys-kernel/assets/164426396/5a99a885-8b53-444a-9264-b16da7f9a368)

Reply:
The community project is just a mountain of shit without strict review.